libs/@local/harpc/net/Cargo.toml

```
@@ -30,30 +30,30 @@ tokio = { workspace = true, public = true, features = ["io-util", "macros"] }
 codec = { workspace = true, features = ["harpc"] }
 error-stack = { workspace = true }
 harpc-wire-protocol = { workspace = true }
+harpc-codec = { workspace = true }
 
 # Private third-party dependencies
 bytes = { workspace = true }
 bytes-utils = { workspace = true }
+derive_more = { version = "1.0.0", features = ["display"] }
 futures = { workspace = true }
 libp2p = { workspace = true, features = ["metrics", "macros", "tcp", "noise", "yamux", "ping", "tokio", "identify"] }
 libp2p-stream = { workspace = true }
 pin-project-lite = { workspace = true }
 scc = { workspace = true }
+serde = { workspace = true, features = ["derive"] }
 tachyonix = { workspace = true }
 thiserror = { workspace = true }
 tokio-stream = { workspace = true, features = ["time"] }
-tokio-util = { workspace = true, features = [
-    "codec",
-    "compat",
-    "rt",
-    "tracing",
-] }
+tokio-util = { workspace = true, features = ["codec", "compat", "rt", "tracing"] }
 tracing = { workspace = true }
 
 [dev-dependencies]
+harpc-codec = { workspace = true, features = ["json"] }
 harpc-types = { workspace = true }
 harpc-wire-protocol = { workspace = true, features = ["test-utils"] }
 humansize = { workspace = true }
+insta.workspace = true
```

Easier to search for dependencies when doing a `<DEP> =` and for consistency:
```suggestion
insta = { workspace = true }
```

---

see: see: [724abfa](https://github.com/hashintel/hash/pull/5362/commits/724abfadf1ad9e7acd51cdbb320921585c0f0f21)